shell/platform/fuchsia/runtime/dart/utils/files.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef TOPAZ_RUNTIME_DART_UTILS_FILES_H_
```

Comment:
Should the include guards be updated to not mention Topaz?

Reply:
Updated.